editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1061,132 +1061,134 @@ void CanvasItemEditor::_switch_theme_preview(int p_mode) {
 	EditorNode::get_singleton()->update_preview_themes(theme_preview);
 }
 
+// Check if we are hovering a guide.
+void CanvasItemEditor::_is_hovering_guide(Point2 p_pos, bool p_is_pressed) {
+	Node *const scene = EditorNode::get_singleton()->get_edited_scene();
+	Array vguides = scene->get_meta("_edit_vertical_guides_", Array());
+	Array hguides = scene->get_meta("_edit_horizontal_guides_", Array());
+	Transform2D xform = viewport_scrollable->get_transform() * transform;
+	int hovered_guide_index = -1;
+	real_t minimum = 1e20;
+
+	if (p_pos.x < RULER_WIDTH) {
+		// Check if we are hovering an existing horizontal guide.
+		for (int i = 0; i < hguides.size(); i++) {
+			if (ABS(xform.xform(Point2(0, hguides[i])).y - p_pos.y) < MIN(minimum, 8)) {
```

Comment:
```suggestion
			if (Math::abs(xform.xform(Point2(0, hguides[i])).y - p_pos.y) < MIN(minimum, 8)) {
```
We generally use the `Math` function for this